File: 自定义视图/code/SlidingConflict/app/src/main/java/com/sliding/harvic/slidingconflict/MainActivity.java
package com.sliding.harvic.slidingconflict;

import android.content.Intent;
import android.support.v7.app.AppCompatActivity;
import android.os.Bundle;
import android.text.method.ScrollingMovementMethod;
import android.view.View;
import android.view.View.OnClickListener;
import android.widget.TextView;

public class MainActivity extends AppCompatActivity {

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_main);


        findViewById(R.id.btn1).setOnClickListener(new View.OnClickListener(){
            @Override
            public void onClick(View v) {
                startActivity(new Intent(MainActivity.this,FirstActivity.class));
            }
        });

        findViewById(R.id.btn2).setOnClickListener(new View.OnClickListener(){
            @Override
            public void onClick(View v) {
                startActivity(new Intent(MainActivity.this,SecondActivity.class));
            }
        });
    }
}

File: OpenGL/code/AndroidOpenGLTutorial-master/MagicCube/src/main/java/com/gluems/magiccube/MainActivity.kt
package com.gluems.magiccube

import android.support.v7.app.AppCompatActivity
import android.os.Bundle
import com.gluems.magiccube.views.MagicCubeView

class MainActivity : AppCompatActivity() {

    lateinit var view: MagicCubeView

    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)

        view = MagicCubeView(this)
        setContentView(view)
    }
}
